renepay on OpenBSD doesn't build

```
cc plugins/renepay/pay.c
In file included from plugins/renepay/pay.c:17:
./plugins/renepay/pay_flow.h:55:3: error: format specifies type 'long' but the argument has type 'u64' (aka 'unsigned long long') [-Werror,-Wformat]
                k->groupid,k->partid,
                ^~~~~~~~~~
./ccan/ccan/tal/str/str.h:43:39: note: expanded from macro 'tal_fmt'
        tal_fmt_(ctx, TAL_LABEL(char, "[]"), __VA_ARGS__)
                                             ^~~~~~~~~~~
In file included from plugins/renepay/pay.c:17:
./plugins/renepay/pay_flow.h:55:14: error: format specifies type 'long' but the argument has type 'u64' (aka 'unsigned long long') [-Werror,-Wformat]
                k->groupid,k->partid,
                           ^~~~~~~~~
./ccan/ccan/tal/str/str.h:43:39: note: expanded from macro 'tal_fmt'
        tal_fmt_(ctx, TAL_LABEL(char, "[]"), __VA_ARGS__)
                                             ^~~~~~~~~~~
plugins/renepay/pay.c:883:7: error: format specifies type 'long' but the argument has type 'u64' (aka 'unsigned long long') [-Werror,-Wformat]
                           last_pending_group_id,
                           ^~~~~~~~~~~~~~~~~~~~~
plugins/renepay/pay.c:885:7: error: format specifies type 'long' but the argument has type 'u64' (aka 'unsigned long long') [-Werror,-Wformat]
                           last_pending_partid);
                           ^~~~~~~~~~~~~~~~~~~
4 errors generated.
gmake: *** [Makefile:300: plugins/renepay/pay.o] Error 1
```

--- a/plugins/renepay/pay.c
+++ b/plugins/renepay/pay.c
@@ -877,9 +877,9 @@ payment_listsendpays_previous(
 
 		plugin_log(pay_plugin->plugin,LOG_DBG,
 			   "There are pending sendpays to this invoice. "
-			   "groupid = %ld, "
+			   "groupid = %"PRIu64" "
 			   "delivering = %s, "
-			   "last_partid = %ld",
+			   "last_partid = %"PRIu64,
 			   last_pending_group_id,
 			   type_to_string(tmpctx,struct amount_msat,&p->total_delivering),
 			   last_pending_partid);

--- a/plugins/renepay/pay_flow.h
+++ b/plugins/renepay/pay_flow.h
@@ -51,7 +51,7 @@ static inline const char* fmt_payflow_key(
 {
 	char *str = tal_fmt(
 		ctx,
-		"key: groupid=%ld, partid=%ld, payment_hash=%s",
+		"key: groupid=%"PRIu64", partid=%"PRIu64", payment_hash=%s",
 		k->groupid,k->partid,
 		type_to_string(ctx,struct sha256,k->payment_hash));
 	return str;

